src/player/player.ts

```diff
@@ -63,64 +67,49 @@ export default class Player<P extends Player<P, B> = any, B extends Game<P, B> =
   /**
    * Returns an array of all other players.
    */
-  others(this: P) {
-    return Array.from(this._players).filter(p => p !== this) as P[];
+  others(): P[] {
+    return Array.from(this._players).filter(p => p as Player !== this);
```

Comment:
wouldn't it make sense to cast this to `P` instead?

Reply:
it actually doesn't matter but I think I had it Player earlier when i was still having trouble with the `this` comparison. Now it's fine to change it tho.